deployment/terraform/assets/cluster.tf

```diff
@@ -271,7 +271,7 @@ EOF
 
 
 resource "aws_rds_cluster_instance" "cluster_instances" {
-  count                      = var.app_instance_count > 0 ? var.db_instance_count : 0
+  count                      = var.db_instance_count
```

Comment:
There was a reason we were checking on `var.app_instance_count`. It's because when we set `AppInstanceCount` to zero (agents only deployment) I'd expect no DB instances to be created. We were defining a dependency relationship between the two values.